apps/portal/controllers/apis/accesstokens.jag

```diff
@@ -31,57 +30,28 @@ function getSamlAccessToken() {
     // match url for patterns
     var matcher = new URIMatcher(request.getRequestURI());
     if (matcher.match('/accesstokens/{id}')) {
-        var vars = matcher.elements();
-        var dashboardId = vars.id;
-        var dashboard = dashboards.getAsset(dashboardId);
-
-        if (dashboard == null) {
-            return "unable to find the dashboard";
-        }
-        var oauthEnabled = dashboard.enableOauth ? true : false;
-
-        if (oauthEnabled) {
-            var provider = {
-                "oauth_version": "2",
-                "authorization_url": "",
-                "access_token_url": dashboard.accessTokenUrl,
-                "request_token_url": "",
-                "callback_url": "",
-                "api_key": dashboard.apiKey,
-                "api_secret": dashboard.apiSecret
-            };
-
-            var wso2is = new oauth.OAuthProvider(provider);
-            var samlResp = session.get("samlResponse");
-            var accessTokenArr = session.get("accessTokenArr");
-            if (accessTokenArr == null) {
-                accessTokenArr = [];
-            }
-            var accessToken;
-            for (var i = 0; i < accessTokenArr.length; i++) {
-                if (accessTokenArr[i].did == dashboardId) {
-                    accessToken = accessTokenArr[i].accessToken;
-                    break;
+        if (tokenUtil.checkOAuthEnabled()) {
+            tokenUtil.getAccessToken(function (accessToken) {
+                if (accessToken == null || accessToken == "") {
+                    log.error("/controllers/apis/accesstokens.jag: Access token has not been set yet");
+                    callBack({"accessToken": ""});
+                } else {
+                    log.warn(accessToken);
```

Comment:
Thanks for pointing out this. I would have missed removing it. It has been removed with
327a22f